scripts/basic-demo.md

```diff
@@ -38,12 +38,13 @@ The 3-party demonstration creates 3 personas with different roles, given below.
 
 ## Running the Demo
 
-The `scripts` folder within the `dscp-hyproof-api` project contains a series of shell scripts (numbered initially with `1` through `4`).
+The `scripts` folder within the `dscp-hyproof-api` project contains a series of shell scripts (numbered initially with `1` through `5`).
 
-`1_load_identities.sh`
-`2_initiate_token.sh`
-`3_add_to_ledger.sh`
-`4_add_eCO2.sh`
+- `1_load_identities.sh`
+- `2_initiate_token.sh`
+- `3_add_to_ledger.sh`
+- `4_add_eCO2.sh`
```

Comment:
should we use lower case only names , e.g. **`4_add_eco2.sh`** ?

Reply:
That would involve changing the name of the script and then where it's described in all other files.

Reply:
ok .. fine :)